server/services/email.js

```diff
@@ -0,0 +1,41 @@
+const sgMail = require("@sendgrid/mail");
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+module.exports = {
+  sendEmail: async ({ to, subject, text }) => {
+    const msg = {
+      to,
+      from: "hatchways.pearl@gmail.com",
+      subject,
+      text,
+    };
+
+    try {
+      await sgMail.send(msg);
+    } catch (error) {
+      console.error(error);
+      if (error.response) {
+        console.error(error.response.body);
+      }
+    }
+  },
+
+  sendEmailWithTemplate: async ({ to, subject, templateId, data }) => {
+    const msg = {
+      to,
+      from: "hatchways.pearl@gmail.com",
+      subject,
+      templateId,
+      dynamic_template_data: data,
+    };
+
+    try {
+      await sgMail.send(msg);
+    } catch (error) {
+      console.error(error);
+      if (error.response) {
+        console.error(error.response.body);
+      }
+    }
+  },
```

Comment:
I would prefer to write each of these functions as a named global function, then use a named export. Doing so will make them more readable, should the function bodies grow in the future.